extension-jaxrs/src/test/java/test/io/smallrye/openapi/runtime/scanner/entities/JaxbWithNameGreeting.java

```diff
@@ -9,11 +12,20 @@
     @XmlAttribute
     private final String message;
 
-    public JaxbWithNameGreeting(String message) {
+    public JaxbWithNameGreeting(String message, List<String> items) {
         this.message = message;
+        this.books = items;
     }
 
     public String getMessage() {
         return message;
     }
+
+    @XmlElementWrapper(name = "books-array")
+    private final List<String> books;
```

Comment:
If you add `@XmlElement(name = "item")`, does the result match the example in the OpenAPI article you linked?

Reply:
Yes. I wanted to add support for XmlElement in the following PR.